doc/stdlib/Library.tex

```diff
@@ -7,7 +7,7 @@
 \usepackage{amsfonts}
 \usepackage{amssymb}
 \usepackage{url}
-\usepackage[color]{../../coqdoc}
+\usepackage[color]{../../tools/coqdoc/coqdoc}
```

Comment:
Is this change because something is not installed in the same place anymore?

Reply:
Umm, not sure, but certainly before the path didn't make a lot of sense?